modules/metastorage-api/src/main/java/org/apache/ignite/internal/metastorage/dsl/SimpleCondition.java

```diff
@@ -29,16 +29,7 @@ public interface SimpleCondition extends Condition {
     ByteBuffer key();
 
     /** Condition type. */
-    int conditionType();
-
-    /**
-     * Returns condition type.
-     *
-     * @return Condition type.
-     */
-    default ConditionType type() {
-        return ConditionType.values()[conditionType()];
-    }
+    ConditionType conditionType();
```

Comment:
It seems logical to leave `type()` as the name of this method: `type()` was returning `ConditionType` before, and the method is defined on a class named `SimpleCondition`, so if it's `type()`, it's already obvious that it's a *condition* type.

Reply:
I probably agree with you, although I'm too lazy to change =)